client/components/list-end/index.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+const listEnd = <div className="list-end" />;
```

Comment:
The eslint error this causes is odd. If there's a better workaround I'm all ears.